CRM/Event/BAO/Event.php

```diff
@@ -2436,15 +2425,23 @@ public static function getICalLinks($eventId = NULL) {
     }
     $return[] = [
       'url' => CRM_Utils_System::url('civicrm/event/ical', $query, TRUE, NULL, TRUE),
-      'text' => $eventId ? ts('Download iCalendar entry for this event.') : ts('Download iCalendar entry for current and future public events.'),
+      'text' => $eventId ? ts('Download iCalendar entry for this event') : ts('Download iCalendar entry for current and future public events'),
```

Comment:
I don't think it's worth breaking translations for this tiny change, seen only in an alt text?

Reply:
Yes, I agree

Reply:
Updated @mlutfy - as an aside would be good to update the translation. It does look a bit odd to have a full stop on this sentence and the sentence itself really should be shorter, like "Add to Calendar"